backend/hackathons/controllers/platforms/devfolioController.js

```
@@ -66,41 +66,41 @@ async function fetchPageWise (from, size, type) {
+      if (totalHackathons > 10) {
         const restOfHackathons = await fetchPageWise(10, totalHackathons, type);
         listOfHackathons.push(...restOfHackathons);
       }
 
       resolve(listOfHackathons);
-    }catch(error){
+    } catch (error) {
       console.log(error);
       resolve([]);
     }
   });
```

Biome (2.1.2)

[error] Promise executor functions should not be `async`.



(lint/suspicious/noAsyncPromiseExecutor)